bin/_mocha

```diff
@@ -98,6 +98,8 @@ program
   .option('--trace', 'trace function calls')
   .option('--trace-deprecation', 'show stack traces on deprecations')
   .option('--watch-extensions <ext>,...', 'additional extensions to monitor with --watch', list, [])
+  .option('--tags <tags>', 'only include tests with the given tags', list, [])
```

Comment:
This version doesn't actually have the `+` syntax, that was just a suggestion in the `gh-pages` branch. But :+1: to the `,`, I'll amend that.